### PR TITLE
sub-zero ascii font

### PR DESCRIPTION
**Pull request by meaghanfitzgerald**
## Why this should be merged

Significantly improves UX

## How this was tested

https://www.patorjk.com/software/taag/#p=display&f=Sub-Zero&t=Avalanche
`./build/avalanchego`

### CODE REVIEW

**Review by yacovm — APPROVED** [review]

File: app/app.go [hunk 1]
```diff
@@ -20,12 +20,13 @@ import (
...
-  /    |    \   /  / __ \|  |__/ __ \|   |  \  \___|   Y  \  ___/    \>> |
-  \____|__  /\_/  (____  /____(____  /___|  /\___  >___|  /\___  >    \\
-          \/           \/          \/     \/     \/     \/     \/`
+const Header = `
+   ______     __   __   ______     __         ______     __   __     ______     __  __     ______    
+  /\  __ \   /\ \ / /  /\  __ \   /\ \       /\  __ \   /\ "-.\ \   /\  ___\   /\ \_\ \   /\  ___\   
+  \ \  __ \  \ \ \'/   \ \  __ \  \ \ \____  \ \  __ \  \ \ \-.  \  \ \ \____  \ \  __ \  \ \  __\   
+   \ \_\ \_\  \ \__|    \ \_\ \_\  \ \_____\  \ \_\ \_\  \ \_\\"\_\  \ \_____\  \ \_\ \_\  \ \_____\ 
+    \/_/\/_/   \/_/      \/_/\/_/   \/_____/   \/_/\/_/   \/_/ \/_/   \/_____/   \/_/\/_/   \/_____/ 
+                                                                                                    `
```
Comment (StephenButtolph) [inline app/app.go:29]:
```suggestion
  _______              ______                   ______       
  ___    |__   _______ ___  /_____ ________________  /______     ,_ o
  __  /| |_ | / /  __ `/_  /_  __ `/_  __ \  ___/_  __ \  _ \    / //\,
  _  ___ |_ |/ // /_/ /_  / / /_/ /_  / / / /__ _  / / /  __/     \>> |
  /_/  |_|____/ \__,_/ /_/  \__,_/ /_/ /_/\___/ /_/ /_/\___/       \\`
```

Comment (StephenButtolph) [inline app/app.go:29]:
```suggestion
    ,---.                  ,--.                      ,--.             
   /  O  \,--.  ,--.,--,--.|  | ,--,--.,--,--,  ,---.|  ,---.  ,---.   ,_ o
  |  .-.  |\  `'  /' ,-.  ||  |' ,-.  ||      \| .--'|  .-.  || .-. :  / //\,
  |  | |  | \    / \ '-'  ||  |\ '-'  ||  ||  |\ `--.|  | |  |\   --.   \>> |
  `--' `--'  `--'   `--`--'`--' `--`--'`--''--' `---'`--' `--' `----'    \\`
```

File: app/app.go [hunk 2]
```diff
@@ -20,12 +20,13 @@ import (
 	"github.com/ava-labs/avalanchego/utils/ulimit"
 )
 
-const Header = `     _____               .__                       .__
-    /  _  \___  _______  |  | _____    ____   ____ |  |__   ____    ,_ o
-   /  /_\  \  \/ /\__  \ |  | \__  \  /    \_/ ___\|  |  \_/ __ \   / //\,
-  /    |    \   /  / __ \|  |__/ __ \|   |  \  \___|   Y  \  ___/    \>> |
-  \____|__  /\_/  (____  /____(____  /___|  /\___  >___|  /\___  >    \\
-          \/           \/          \/     \/     \/     \/     \/`
+const Header = `
```
Comment (StephenButtolph) [inline app/app.go:23]: The skier must stay - for significant UX reasons.

**Review by StephenButtolph — CHANGES_REQUESTED** [review]

**Comment by StephenButtolph** [comment]
Also we should keep the banner <= 80 characters for best terminal UX

File: app/app.go (hunk 2 above)
Comment (meaghanfitzgerald) [reply app/app.go:23]: dnlgtm

**Pull request closed without merging**